tnnt/views.py

```diff
@@ -1,4 +1,18 @@
 from django.views.generic import TemplateView
+from django.views import View
+from django.shortcuts import render
+from scoreboard.models import Player, Clan
+from tnnt.forms import CreateClanForm, InviteMemberForm
+from django.http import HttpResponseRedirect
+from . import hardfought_utils # find_player
+
+def impossible(*args):
+    # Yes, I made impossible(). I'm too used to having it around.
```

Comment:
I like having this impossible() but would suggest maybe having this additionally call die/exit or whatever, so that it behaves like a panic.
Also, there in the comment you say it's useful for seeing where something “really is known to be impossible” which I would reword as something like “catching things that *should* be impossible but aren't.”

Reply:
I don't think we should crash the server and bring it down for *everyone* when someone manages to trigger one of these impossible cases (some of which can intentionally be done with a malicious POST request).